### PR TITLE
Fix type definition of `Event` constructor

### PR DESCRIPTION
**Pull request by srenault**

### CODE REVIEW

File: src/lib/dom.generated.d.ts [hunk 1]
```diff
@@ -7344,7 +7344,7 @@ interface Event {
 }
 declare var Event: {
     prototype: Event;
-    new(): Event;
+    new(type: string): Event;
```
Comment (RyanCavanaugh) [inline src/lib/dom.generated.d.ts]: :+1:

This constructor throws if no name is provided, so this parameter should be considered required.

**Comment by mhegazy** [comment]
@srenault thanks for the PR. the issue is that this file is actually auto generated from the IE definition files. this makes it easy for us to get an almost accurate DOM definitions without having to update them manually. We plan on updating how the dom typings are generated but for now, we can not make hand edits to this file. If you can log a separate issue, i can get it fixed in our script and update the file.

**Comment by srenault** [comment]
Ok, I suspected this. It there anyway in my project to override / add a definition of Event constructor without edit lib.d.ts ?

**Comment by mhegazy** [comment]
I can not think of one. @RyanCavanaugh ideas?

**Pull request closed without merging**